### ISSUE TITLE
Also discuss ChatGPT and other AI

### ISSUE THREAD

**Issue by erikvansebille** [SanliFaez/OS4Physicists#51]
As @veldersguus suggested, maybe we should also discuss the role of ChatPGT and similar AI in the scientific practice?

**Comment by erikvansebille** [comment]
I could see AI fit in Week 4 (reproducibility) or Week 8 (the scientific process). @SanliFaez, what do you think?

**Comment by SanliFaez** [comment]
not sure yet! the technical part is not really fit for this course, the ethical part perhaps a bit but I am not sure how to fit it into the course. I am not that good in using it for code standardization and auto-checks. that latter can be an area to talk abotu.

**Comment by erikvansebille** [comment]
I agree that the technicalities of LLMs are not within scope for this course; but the ethics could fit in as the role of Machines in a team? We could bring it up with the guest speaker of Week 3?

**Comment by veldersguus** [comment]
Hi Erik  and  Sanli,


My idea was also to address the practical use of Chatbots and if or how students can use it. What they have to think if when using it.


For the MAIO course (alsom in blok 1) the students have to apply a machine learning technique to analyse data. They also have to write a report/paper about it. I am think of allowing them to use a chatbot, but want to learn them (by their experience) what the strength and weaknesses are.


Best,

Guus


________________________________

**Comment by SanliFaez** [comment]
I think, in the new edition of the course, this topic will naturally fit in the lecture on knowledge discovery. Would you like consider this tip by Guus @thinderer ?
